charts/kubearchive/templates/sink.yaml

```
@@ -17,6 +17,8 @@ spec:
       containers:
         - name: {{ .Values.sink.name }}
           image: {{ .Values.sink.image }}
+          env:
+{{ include "kubearchive.v1.otel.env" .Values.sink | indent 12 }}
```

I think we are hiding information in some way by doing it this way. I think that a more explicit approach works better:

```suggestion
          env:
          - name: KUBEARCHIVE_OTEL_ENABLED
            value: {{ .Values.sink.observability }}
```

While extracting the environment to a template is useful for very long environment scenarios, I don't think we are there yet, and we may never be.

---

basically all configuration for otel instrumentation is done through environment variables. I haven't gotten to the point where we are deploying a collector, but that's the next thing on my list and having a template to make it easy to add the env vars to both the sink and the api server will make configuring the instrumentation for both components a lot easier to manage